[docs] Expand the EAS Metadata documentation (#19028)
expo/expo: docs/pages/eas/metadata/getting-started.md

> Using VS Code? Install the [VS Code Expo plugin](https://github.com/expo/vscode-expo#readme) for auto-complete, suggestions, and warnings in your **store.config.json** files.

## Create the store config

Let's start by creating our **store.config.json** file in the root directory of your project. This file holds all the information you want to upload to the app stores.

If you already have an app in the stores, you can pull the information into a store config by running:

<Terminal cmd={['$ eas metadata:pull']} />

If you don't have an app in the stores yet, EAS Metadata can't generate the store config for you. Instead, create a new store config file.

<CodeBlocksTable tabs={['store.config.json']}>

```json
{
  "configVersion": 0,
  "apple": {
    "info": {
      "en-US": {
        "title": "Awesome App",
        "subtitle": "Your self-made awesome app",
        "description": "The most awesome app you've ever seen",
        "keywords": ["awesome", "app"],
        "marketingUrl": "https://example.com/en/promo",
        "supportUrl": "https://example.com/en/support",
        "privacyPolicyUrl": "https://example.com/en/privacy"
      }
    }
  }
}
```

</CodeBlocksTable>

> > By default, EAS Metadata uses the **store.config.json** file in the root of your project. You can change the name and location of this file by setting the **eas.json** [`metadataPath`](../../submit/eas-json.md#metadatapath) property.

Reviewer: ```suggestion
> By default, EAS Metadata uses the **store.config.json** file at the root of your project. You can change the name and location of this file by setting the **eas.json** [`metadataPath`](../../submit/eas-json.md#metadatapath) property.
```

## Update the store config

Now it's time to edit the **store.config.json** file and customize it to your app needs. You can find all available options in the [store config schema](./schema.md).

## Upload a new app version

> Before pushing the **store.config.json** to the app stores, you must upload a new binary of your app. [Read more about uploading new binaries to stores](../../submit/introduction.md).

Reviewer: ```suggestion
Before pushing the **store.config.json** to the app stores, you must upload a new binary of your app. For more information, see [uploading new binaries to stores](../../submit/introduction.md).
```
